kube/workspace/workspace-deployment.yaml

```diff
@@ -48,7 +48,7 @@ spec:
             - name: TRANSACTOR_URL
               value: ws://transactor:3333;ws://localhost:3333
             - name: NOTIFY_INBOX_ONLY
```

Comment:
Could you please remove it at all? This env variable is not used anymore

Reply:
Done